packages/block-library/src/navigation-menu/edit.js

```diff
@@ -60,19 +60,21 @@ function NavigationMenu( {
 		[ pages ]
 	);
 
-	const navigationMenuStyles = {};
-	if ( textColor.color ) {
-		navigationMenuStyles[ '--color-menu-link' ] = textColor.color;
+	const navigationMenuInlineStyles = {};
+	if ( attributes.customTextColor) {
+		navigationMenuInlineStyles[ 'color' ] = attributes.customTextColor;
 	}
 
-	if ( backgroundColor.color ) {
-		navigationMenuStyles[ '--background-color-menu-link' ] = backgroundColor.color;
+	if ( attributes.customBackgroundColor ) {
```

Comment:
> customTextColor and customBackgroundColor don't seem to be coming through at all in editor mode.
it does, when it sets custom colors:

<img src="https://user-images.githubusercontent.com/77539/68160287-35f1d680-ff32-11e9-96ae-19a20edd3115.png" width="400px" />

<img src="https://user-images.githubusercontent.com/77539/68160255-1b1f6200-ff32-11e9-9875-519ddaf876ce.png" width="400px" />

> Could we use textColor and backgroundColor instead?

`textColor` and `backgroundColor` are to apply the styles but with CSS classes instead of inline styles.

Reply:
Handled here: https://github.com/WordPress/gutenberg/pull/18172/commits/91a507ed3ad408b8f5ec75409fad312015cfa2b3 🙇